index.html

```diff
@@ -0,0 +1,552 @@
+        to move to the Web platform. It gives them access to the PC/SC
+        implementation (and card reader drivers) available in the host OS.
+
+        There is also a companion <a href="https://github.com/WICG/web-smart-card/blob/main/README.md">explainer</a> document.
+    </section>
+      </p>
+    </section>
+    <section id='sotd'>
+      <!-- This section is filled automatically by ReSpec. -->
+    </section>
+
+    <section data-dfn-for="Navigator">
+      <h2>Extensions to the {{Navigator}} interface</h2>
+      <pre class="idl">
+        [Exposed=Window, SecureContext]
+        partial interface Navigator {
+          [SameObject] readonly attribute SmartCardResourceManager smartCard;
+        };
+      </pre>
+
+      <h3><dfn>smartCard</dfn> attribute</h3>
+      <p>When getting, the {{Navigator/smartCard}} attribute always returns the same
+      instance of the {{SmartCardResourceManager}} object.</p>
+    </section>
+
+    <section data-dfn-for="WorkerNavigator">
+      <h2>Extensions to the {{WorkerNavigator}} interface</h2>
+      <pre class="idl">
+        [Exposed=DedicatedWorker, SharedWorker, SecureContext]
+        partial interface WorkerNavigator {
+          [SameObject] readonly attribute SmartCardResourceManager smartCard;
+        };
+      </pre>
+
+      <h3><dfn>smartCard</dfn> attribute</h3>
+      <p>When getting, the {{Navigator/smartCard}} attribute always returns the same
+      instance of the {{SmartCardResourceManager}} object.</p>
+    </section>
+
+    <section data-dfn-for="SmartCardResourceManager">
+      <h2><dfn>SmartCardResourceManager</dfn> interface</h2>
+      <pre class="idl">
+        [Exposed=(DedicatedWorker, SharedWorker, Window), SecureContext]
+        interface SmartCardResourceManager {
+          Promise&lt;SmartCardContext&gt; establishContext();
+        };
+      </pre>
+      <section>
+        <h3><dfn>establishContext()</dfn> method</h3>
+        <p>Requests a PC/SC context from the platform's PC/SC stack.</p>
+      </section>
+    </section>
+
+    <section data-dfn-for="SmartCardContext">
+      <h2><dfn>SmartCardContext</dfn> interface</h2>
+      <p>A context for communicating with the PC/SC resource manager.</p>
+      <pre class="idl">
+        [Exposed=(DedicatedWorker, SharedWorker, Window), SecureContext]
+        interface SmartCardContext {
+          Promise&lt;sequence&lt;DOMString&gt;&gt; listReaders();
+
+          Promise&lt;sequence&lt;SmartCardReaderStateOut&gt;&gt; getStatusChange(
+              sequence&lt;SmartCardReaderStateIn&gt; readerStates,
+              AbortSignal signal);
+
+          Promise&lt;SmartCardConnectResult&gt; connect(
+              DOMString readerName,
+              SmartCardAccessMode accessMode,
+              optional sequence&lt;SmartCardProtocol&gt; preferredProtocols);
+        };
+      </pre>
+      <section>
+        <h3><dfn>listReaders()</dfn> method</h3>
```

Comment:
Just for tracking purposes, for this and other methods where there is no algorithm defined please add:

```html
<div class="issue">Write an algorithm for this method.</div>
```

Reply:
Good idea. Done.